Description: Fix failure to build with GCC 4.9
 gcc-4.9 doesn't like the conflict between two variables named
 "error", so rename one of them to avoid the conflict.
Author: Sophie Brun <sophie@freexian.com>
Origin: vendor
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=746915
Last-Update: 2014-08-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/SSLConnectionManager.cpp
+++ b/SSLConnectionManager.cpp
@@ -95,13 +95,13 @@ void SSLConnectionManager::interceptUpda
   } catch (SSLConnectionError &error) {
     std::stringstream errorStream;
     errorStream << "Got exception: " << error.what();
-    std::string error = errorStream.str();    
-    Logger::logError(error);
+    std::string errortext = errorStream.str();
+    Logger::logError(errortext);
   } catch (FirefoxUpdateException &error) {
     std::stringstream errorStream;
     errorStream << "Got exception: " << error.what();
-    std::string error = errorStream.str();    
-    Logger::logError(error);
+    std::string errortext = errorStream.str();
+    Logger::logError(errortext);
   }
 }
 
@@ -120,13 +120,13 @@ void SSLConnectionManager::interceptAddo
   } catch (SSLConnectionError &error) {
     std::stringstream errorStream;
     errorStream << "Got exception: " << error.what();
-    std::string error = errorStream.str();    
-    Logger::logError(error);
+    std::string errortext = errorStream.str();
+    Logger::logError(errortext);
   } catch (FirefoxUpdateException &error) {
     std::stringstream errorStream;
     errorStream << "Got exception: " << error.what();
-    std::string error = errorStream.str();    
-    Logger::logError(error);
+    std::string errortext = errorStream.str();
+    Logger::logError(errortext);
   }
 }
 
@@ -151,9 +151,9 @@ void SSLConnectionManager::interceptSSL(
     } catch (SSLConnectionError &error) {
       std::stringstream errorStream;
       errorStream << "Got exception: " << error.what();
-      std::string error = errorStream.str();
+      std::string errortext = errorStream.str();
 
-      Logger::logError(error);
+      Logger::logError(errortext);
     }
   }
 }
